backend/public-graph/graph/resolver.go

```diff
@@ -2316,9 +2316,30 @@ func (r *Resolver) ProcessBackendPayloadImpl(ctx context.Context, sessionSecureI
 
 	querySessionSpan.Finish()
 
+	var project model.Project
+	if err := r.DB.Where(&model.Project{Model: model.Model{ID: projectID}}).First(&project).Error; err != nil {
+		log.WithContext(ctx).WithError(err).WithField("project", project).WithField("projectVerboseID", projectVerboseID).Error("failed to find project")
+	}
+
 	// Filter out empty errors
 	var filteredErrors []*publicModel.BackendErrorObjectInput
 	for _, errorObject := range errorObjects {
+
+		// Filter out by project.ErrorFilters, aka regexp filters
+		var err error
+		matchedRegexp := false
+		for _, errorFilter := range project.ErrorFilters {
+			matchedRegexp, err = regexp.MatchString(errorFilter, errorObject.Event)
+			if err != nil {
+				log.WithContext(ctx).WithField("regex", errorFilter).WithError(err).Error("invalid regex: failed to parse backend error filter")
```

Comment:
Should we have any frontend validation on whether the regex is valid? This is going to silently fail for the end user and they'll have no idea.